pkg/k8s/pod/converter.go

```diff
@@ -56,6 +56,7 @@ func (c *PodConverter) ConvertList(originalList interface{}) (convertedList inte
 		},
 	}
 	for _, pod := range podList.Items {
+		pod := pod // Fix gosec G601, so we can use &node
```

Comment:
I am not sure if we want to fix this for now since golang 1.22 changes the variable in for loop. https://tip.golang.org/doc/go1.22

Or can you create a backlog to revisit this when we upgrade to v1.22? I think we will upgrade soon.

Reply:
Offline syncing with @sushrk. We can have a backlog to revisit this change after upgrading to go 1.22.